src/resolvers/Mutation/removeFund.ts

```diff
@@ -67,7 +67,7 @@ export const removeFund: MutationResolvers["removeFund"] = async (
     .select("organizationId")
     .lean();
 
-  const currentOrgId = currentOrg?.organizationId?.toString() || "";
+  const currentOrgId = currentOrg?.organizationId?.toString();
```

Comment:
Why is this changed and not the test?

Reply:
This line was missing test coverage, and the unnecessary string concatenation was discovered while looking at the code. 
Although the existing tests passed, so there was no need to change the test file or add any tests.